website/src/main/tut/docs/rules/ImplicitClassPrivateVal.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: docs
+title: ImplicitClassPrivateVal
+---
+
+# ImplicitClassPrivateVal
+
+`val` fields of implicit classes are accessible as an extension. 
+This rule adds the `private` access modifier in such cases in order to prevent direct access.
+
+```scala
+// before
+implicit class XtensionVal(val str: String) extends AnyVal {
+  def doubled: String = str + str
+}
+"message".str // compiles
+
+// after
+implicit class XtensionValFixed(private val str: String) extends AnyVal {
+  def doubled: String = str + str
+}
+"message".str // does not compile
+```
+
+This rule safely ignores all other modifiers.
```

Comment:
Note. This rule only triggers for `val` fields, it ignores other modifiers such as `var`.